packages/kotti-navbar/src/KtNavbar.vue

```diff
@@ -94,15 +98,12 @@ export default {
 		labelText: { type: String, default: null },
 	},
 	inject: {
-		themeColor: {
-			from: 'KtNavbarTheme',
-			default: () => ({
-				backgroundColor: '#122C56',
-				textColor: 'rgba(255,255,255, 0.8)',
-				activeColor: 'rgba(255,255,255, 1)',
-				borderColor: '#rgba(255,255,255, 0.14)',
-				logoUrl: null,
-			}),
+		KtTheme: {
+			default: {
+				navbarBackgroundColor: '#122C56',
```

Comment:
I’d suggest moving this (partially repeated, and logically belonging together) piece of code to a utility file (e.g. `default-theme.json`). You can just import that everywhere where you need the default theme then. 🙂

Reply:
yep, a good idea. can do when we have more theme need to be configured.